devices/pine64-pinephonepro/default.nix

```
@@ -0,0 +1,123 @@
+{ config, lib, pkgs, ... }:
+{
+  mobile.device.name = "pine64-pinephonepro";
+  mobile.device.identity = {
+    name = "Pinephone Pro";
+    manufacturer = "Pine64";
+  };
+
+  boot.kernelParams = [
+    # Serial console on ttyS2, using the dedicated cable.
+    "console=ttyS2,115200"
+    "earlycon=uart8250,mmio32,0xff1a0000"
+    "earlyprintk"
+
```

```suggestion
    # Serial console on ttyS2, using the dedicated cable.
    # TODO: option to enable serial console.
    #"console=ttyS2,115200"
    #"earlycon=uart8250,mmio32,0xff1a0000"
    #"earlyprintk"

```

For now comment these out like done e.g. here:

 - https://github.com/NixOS/mobile-nixos/blob/4d2093efa7efa00131d385fd9d11e54ce16bc57a/devices/lenovo-wormdingler/default.nix#L41-L44

Add the `TODO` comment as well. Eventually (haha) someone (me?) will come back around and add an option to describe the required config for the serial console, which will be controlled behind an `.enable` option.